src/index.html

```diff
@@ -5,10 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+       href="https://fonts.gstatic.com"
+       crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"><img src="images/logo.png" alt="Moyo logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
```

Comment:
```suggestion
            <a
              href="#laptops&amp;computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
Don't see the difference. What should I change?

Reply:
@yuliiababych spaces
<img width="354" alt="Screenshot at Aug 22 11-30-49" src="https://github.com/mate-academy/layout_moyo-header/assets/77551820/d2844588-3372-4a69-bba9-28e23383c305">